cedar-policy-validator/src/schema_file_format.rs

```diff
@@ -78,6 +78,19 @@ impl NamespaceDefinition {
             actions: actions.into_iter().collect(),
         }
     }
+
+    pub fn merge_actions_by_type(&self) -> Vec<(&ActionType, HashSet<SmolStr>)> {
```

Comment:
doc string? also why return a `Vec` of pairs instead of a `HashMap`?

Reply:
I don't think this function is used anywhere now.

Reply:
Removed it